package.json

```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
```

Comment:
- [ ] Kindly deploy the project online and include the live deploy link in the readme.